Utils/utils/password_generator.py

```diff
@@ -0,0 +1,29 @@
+import secrets
+import string
+from sekoia_automation.action import Action
+
+
+class PasswordGenerator(Action):
+    def run(self, arguments: dict):
+        password_length = self.json_argument("password_length", arguments)
+        number_of_digits = self.json_argument("number_of_digits", arguments)
+        number_of_special_characters = self.json_argument("number_of_special_characters", arguments)
```

Comment:
No need to fetch it from a file also setting defaults would be nice
```suggestion
        password_length = arguments.get("password_length", 20)
        number_of_digits = arguments.get("number_of_digits", 1)
        number_of_special_characters = arguments.get("number_of_special_characters", 1)
```